react/features/settings/components/native/SettingsView.tsx

```diff
@@ -140,23 +164,35 @@ class SettingsView extends AbstractSettingsView<Props, State> {
      *
      * @inheritdoc
      */
-    constructor(props) {
+    constructor(props: Props) {
         super(props);
+
         const {
+            // @ts-ignore
```

Comment:
Instead of all these ignores change the Props interface. For settings instead of Object use the actual object ({disableCallIntegration: boolean; etc})

Reply:
Nice. Thanks for the hint!